src/components/10-atoms/progress-bar/README.md

```diff
@@ -0,0 +1,35 @@
+# Progress Bar
+
+A progress bar that can be used for when something is loading and when you can determind the progress.
+
+## Properties
+
+### Variant
+
+| Attribute   | Details                                                                       |
+| ----------- | ----------------------------------------------------------------------------- |
+| `small`     | Smaller version of the progress bar (boolean, default: false)                 |
+| `fullWidth` | Removes the `border-radius`so it can fit to an edge (boolean, default: false) |
+| `value`     | Sets the value (or progress) of the loader.                                   |
+| `max`       | Sets the max value of the progress bar (default: 100).                        |
+| `text`      | Sets the text below the progress bar.                                         |
+
+### small
+
+Smaller version of the progress bar (boolean, default: false)
+
+### fullWidth
+
+Removes the `border-radius`so it can fit to an edge (boolean, default: false)
+
+### value
+
+Sets the value (or progress) of the loader. The `value` can not get over 100
```

Comment:
"The value can not surpass 100."

Is this true? Sounds wrong, because I can set Max to 9999 and then the value can go higher than 100 right?

Therefore, my new recommendation:
"Sets the progress state of the progress bar. `value` can not surpass the value of `max`."